### ISSUE TITLE
Doesnt work on discord

### ISSUE THREAD

**Issue by RuLasky** [AllanCat/PS3EyeDirectShow#1]
Please need help :(

**Comment by AllanCat** [comment]
Need further information, which part is not working? Did it work on other apps like chrome?
Did you follow the [installation](https://github.com/AllanCat/PS3EyeDirectShow#installation) and make sure driver is correct?
It should be USB Camera-B4.09.21.1 in device manager like this
![image](https://github.com/AllanCat/PS3EyeDirectShow/assets/1924514/ec808fbf-4e65-4b7b-b07b-39dd40447831)

**Comment by RuLasky** [comment]
Yes I followed the instructions, it appears fine in opentrack and obs but
remains black on discord. Using the virtual camera it works but not
directly without

**Comment by AllanCat** [comment]
I did encountered discord black screen once, but couldn't reproduce the issue after reboot.
DebugView shown it stopped after "initing device", I think it might relate to webcam was still being used or list_devices() didn't return correct device numbers.

In short: try reboot and see if it fix the problem.

**Comment by PlayerLoler** [comment]
The new version [Universal PS3 Eye Driver 1.1](https://github.com/AllanCat/PS3EyeDirectShow/releases/tag/1.1) seems to have fixed the issue
